runtests.jl

```diff
@@ -18,7 +18,7 @@ function run_solution() end
 
 function run_solution(source_path, input_path, ::Val{:cpp})
     exename = replace(splitpath(source_path)[end], "cpp" => "o")
-    run(`g++ -std=c++20 $source_path -o $exename`)
+    run(`g++ -std=c++23 $source_path -o $exename`)
```

Comment:
wow, living in the future

Reply:
even c++23 doesn't have the function I wanted, but another way was used to implement the transform lambda function and solved the problem.

Reply:
do you have access? https://github.com/RongkunWang/AoC2023/actions/runs/7072796215/job/19252036397#step:6:123

Reply:
yes. that's quite an error...
```cpp
error: no matching function for call to ‘std::ranges::split_view<std::ranges::ref_view<const std::__cxx11::basic_string<char> >, std::ranges::single_view<char> >::_OuterIter<false>::value_type::data()’
   13 |     std::string_view s (v.data(), v.data() + v.size());
      |                         ~~~~~~^~
```

Reply:
I'll take a look tonight

Reply:
Is this related to https://stackoverflow.com/questions/56118941/do-we-have-c20-ranges-library-in-gcc-9 ? I know it looks like it's just one function and not the whole library missing.

Reply:
Ah, so maybe we need a newer gcc?